src/entities/amm.ts

```diff
@@ -1705,51 +1705,19 @@ class AMM {
 
   // underlying token approval for periphery
 
-  public async isUnderlyingTokenApprovedForPeriphery(): Promise<boolean | void> {
```

Comment:
This function is used by the current UI so we need to make that we push the change in there as well.

Reply:
Indeed. UI change to follow. Are we happy to make this non-back-compatible change and increment the major version of the SDK, or should I retain some back-compatible functions?

Reply:
Indeed. The scenario we need to make sure doesn't happen is:
   - merge this without updating UI
   - merge other SDK work [x] on top of that
   - bumping SDK to UI at later time and adjusting UI for [x] but failing because we haven't adjust for this.
   
   If we're ready to merge the work on the UI for this SDK change, I'm happy.